src/Asset/Capacity/AbstractCapacity.php

```diff
@@ -65,6 +66,28 @@ public function getSpecificRights(): array
         return [];
     }
 
+    public function canBeDisabled(CommonDBTM $asset, string $table = null): bool
+    {
+        if ($table !== null) {
+            /** @var \DBmysql $DB */
+            global $DB;
+
+            return $DB->request(
+                $table,
+                [
+                    'itemtype' => 'Glpi\CustomAsset\\' . $asset->fields['system_name'],
+                ]
+            )->count() == 0;
```

Comment:
You can use the `countElementsInTable()` global function. Anyway, when you want to get a result count only, please use the a `SELECT COUNT` query that will be more efficient.